zoo-mysql-dal/src/grpc-tree/tasks/messages/delete.proto

```
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package dal.tasks;
+
+import "common/messages/types.proto";
```

Buf (1.54.0)

import "common/messages/types.proto": file does not exist

(COMPILE)